posthog/api/test/test_event.py

```diff
@@ -209,16 +213,20 @@ def test_filter_by_person_for_person_on_events_enabled(self, mock_actor_on_event
 
         flush_persons_and_events()
 
-        response = self.client.get(f"/api/projects/{self.team.id}/events/?person_id={initial_person.uuid}").json()
+        call_endpoint = lambda self: self.client.get(
+            f"/api/projects/{self.team.pk}/events/?person_id={initial_person.uuid}"
+        ).json()
+        response = snapshot_clickhouse_queries(call_endpoint)(self)
```

Comment:
I did this to avoid having all of the calls produce query snapshots, but I'll switch to the other, it's a pretty small query anyway so shouldn't be too unwieldy

Reply:
Have updated now 🤞